data/component-design-tokens/date-picker-design-tokens.json

```
@@ -20,8 +34,14 @@
     "type": "spacing"
   },
   "date-input-day-font-today": {
```

Today's date font should be heading/2xs


```suggestion
 "date-input-day-font-today": {
      "value": "{typography.heading.2xs}",
      "type": "other"
    },
```